### ISSUE TITLE
Move codebase to the new WebEngine

### ISSUE THREAD

**Issue by navrocky** [MiT-Ufa/Slacker#7]
This is a required step cause WebKit already deprecated for a long time

**Comment by Snegovikufa** [comment]
Yeah, I tried to rewrite TeamWords using Qt 5.6 but seems that WebEngine does not support webkit plugins that are needed for notifications

**Comment by Snegovikufa** [comment]
@navrocky if you share some information about notifications in QtWebEngine you will help a lot

**Comment by Snegovikufa** [comment]
https://trello.com/c/SEwvMCfx/225-implement-desktop-notifications-on-the-qtwebengine-side

**Comment by navrocky** [comment]
I think we can inject self implemented Notification object to target site and thus handle notifications

https://www.w3.org/TR/notifications/#api

To interact with self implemented Notification object we can use [QWebChannel](http://doc.qt.io/qt-5/qwebchannel.html)
